README.md

```diff
@@ -603,72 +567,107 @@ a similar concept (variable reference immutability) while this proposal intends
-- Value type classes
-- ConstSet and ConstMap, the const versions of [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) and [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)
+We are developing the deep path properties proposal as a separate follow-on proposal because we don't see it as core to using Records, which work well independently. It's the kind of syntactic addition which would work well to prototype over time in transpilers, and where we have many decision points which don't have to do with Records and Tuples (e.g., how it works with objects).
 
-A goal of the broader set of proposals (including [operator overloading](https://github.com/littledan/proposal-operator-overloading/) and [extended numeric literals](https://github.com/tc39/proposal-extended-numeric-literals) is to provide a way for user-defined types to do the same as [BigInt](https://github.com/tc39/proposal-bigint).
+## Could I "box" a pointer to an object, and put that in a Record or Tuple?
 
-If value type classes are standardized, features like [Temporal Proposal](https://github.com/tc39/proposal-temporal) which might be able to express its types using const classes. However, this is far in the future, and we do not encourage people to wait for the addition of const classes.
+Maybe! It's unclear exactly how such a box would work. We've been thinking about this in the [Boxing Objects](https://github.com/rricard/proposal-boxing-objects) proposal repo, but this is very early, and still undergoing change. One alternative is that we could permit JS developers to use Symbols as WeakMap keys, so that they could build their own way to use Symbols as such a box.
 
-A previous version of this proposal included `with` syntax for creating new const value types. The community has suggested alernative syntax that could be
-considered in a follow up proposal.
+Overall, we believe Records and Tuples are sufficiently useful on their own even without a built-in box mechanism. See the [Boxing Objects README](https://github.com/rricard/proposal-boxing-objects) for further explanation.
 
-- ["Set-builder inspired notation"](https://github.com/rricard/proposal-const-value-types/issues/44)
+## How does this relate to the [Readonly Collections](https://github.com/tc39/proposal-readonly-collections) proposal?
 
-# Glossary
+We've talked with the Readonly Collections champions, and both groups agree that these are complements:
+- Readonly collections are shallowly immutable and may point to objects; they may be mutated during construction, and read-only views of mutating objects are supported.
+- Records and Tuples are deeply immutable and consist just of primitives.
 
-#### Value type
+Neither one is a subset of the other in terms of functionality. At best, they are parallel, just like each proposal is parallel to other collection types in the language.
```

Comment:
I would use the term "orthogonal" instead of "parallel", "parallel has the connotation it does the same thing but differently

Reply:
I think "orthogonal" is a gratuitously confusing word for most people.

Reply:
probably but parallel is also confusing, can we find another word?

Reply:
"independent"?